samples/test/specs/web-app.js

```diff
@@ -19,16 +19,16 @@ const sampleConfig = getSampleConfig();
 describe('web-app: ' + sampleConfig.name, () => {
 
   // TODO: fix this test. Fails in incognito, succeeds in regular window. May be related to 3rd party cookies.
-  // xit('can login directly, calling signin() with username and password', async () => {
-  //   await startApp(WebApp);
-  //   await loginDirect(WebApp);
-  //   if (sampleConfig.oidc) {
-  //     await browser.pause(5000);
-  //     await WebApp.assertAccessToken();
-  //   } else {
-  //     await WebApp.assertLoginSuccess();
-  //   }
-  // });
+  it('can login directly, calling signin() with username and password', async () => {
```

Comment:
Does the thing described in the `TODO` comment above still exist? If not, we can remove the todo comment.